src/schema/searchableItem.ts

```diff
@@ -38,7 +41,22 @@ export const SearchableItem = new GraphQLObjectType<any, ResolverContext>({
     },
     description: {
       type: GraphQLString,
-      resolve: item => item.description,
+      resolve: ({ description, end_at, label, published_at, start_at }) => {
+        switch (label) {
+          case "Article":
+            return moment.utc(published_at).format("MMM Do, YYYY")
```

Comment:
I thought the description for articles was this date, but only prepended to the actual description field? Now you're only returning the date, but dropping the description.

Reply:
Ahh, I lost some of the copy around the date here. I'll add it back in.

Reply:
Oh, and I didn't realize that we were actually getting descriptions from Elasticsearch, but you're totally right, we should join the two together.